settings.gradle

```diff
@@ -1,4 +1,32 @@
+pluginManagement {
+    repositories {
+        resolutionStrategy {
+            eachPlugin {
```

Comment:
please put a comment on why we're doing this. It looks suspicious, but I don't know enough gradle to put my finger on it.

Reply:
Yeah, this is a funky thing for testrunner. I'll add a comment to that effect.